package.json

```diff
@@ -2,7 +2,7 @@
   "name": "wes-workforce-profiles",
   "version": "0.1.0",
   "private": true,
-  "homepage": "https://plotandscatter.com/wes-workforce-profiles",
```

Comment:
reminds me of forgetting your keys as you walk out the door. :smile:

Reply:
😅